quickstart/aspnet/ContosoTeamStats/RedisConnection.cs

```diff
@@ -143,14 +143,16 @@ private async Task ForceReconnectAsync(bool initializing = false)
                 _firstErrorTime = DateTimeOffset.MinValue;
                 _previousErrorTime = DateTimeOffset.MinValue;
 
-                ConnectionMultiplexer oldConnection = _connection;
-                try
-                {
-                    await oldConnection?.CloseAsync();
```

Comment:
This bad pattern appears in 4 places in this repo. Please scan all files for potential variations where we're awaiting something that's potentially null. 

quickstart\aspnet\ContosoTeamStats\RedisConnection.cs:                    await oldConnection?.CloseAsync();
quickstart\aspnet-core\ContosoTeamStats\RedisConnection.cs:                    await oldConnection?.CloseAsync();
quickstart\dotnet\Redistest\RedisConnection.cs:                    await oldConnection?.CloseAsync();
quickstart\dotnet-core\RedisConnection.cs:                    await oldConnection?.CloseAsync();